bot.ml

```diff
@@ -545,7 +541,7 @@ let job_action json =
     |> Lwt.async
   else if String.equal build_status "success" then (
     (fun () ->
-      get_status_check ~repo_full_name ~commit ~context
+      get_status_check ~repo_full_name ~commit ~context ~header_list:gitlab_header
```

Comment:
The `github_header` should be included in `header_list` too, right?

Reply:
No, the gitlab api is only used in this function call, so `gitlab_header` is not needed.